docs/learn/learn-proxies.md

```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
 
 :::
 
-## Why use a Proxy?
+## Proxy calls 
+
+Proxy calls are used by proxies to call proxied accounts. These calls are especially important for anonymous proxies, as any attempt to sign transactions with an anonymous proxy will fail. For more details see the [dedicated section about anonymous proxies](#anonymous-proxies).
+
+:::info Walk-through tutorial video
+
+You can see this video tutorial that goes through the scenarios below. The tutorial requires some familiarity with the Extrinsic Tab of the Polkadot-JS UI.
```

Comment:
Add the relevant video link here

Reply:
I will produce those short videos very soon